[2021-12-27 11:33:32,866] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_kildare.scrap_data scheduled__2021-12-26T09:45:00+00:00 [queued]>
[2021-12-27 11:33:33,114] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_kildare.scrap_data scheduled__2021-12-26T09:45:00+00:00 [queued]>
[2021-12-27 11:33:33,131] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-27 11:33:33,145] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-27 11:33:33,152] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-27 11:33:33,255] {taskinstance.py:1262} INFO - Executing <Task(SparkSubmitOperator): scrap_data> on 2021-12-26 09:45:00+00:00
[2021-12-27 11:33:33,300] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_kildare', 'scrap_data', 'scheduled__2021-12-26T09:45:00+00:00', '--job-id', '4808', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_kildare.py', '--cfg-path', '/tmp/tmpye8sckx5', '--error-file', '/tmp/tmpc8z4lif2']
[2021-12-27 11:33:33,312] {standard_task_runner.py:77} INFO - Job 4808: Subtask scrap_data
[2021-12-27 11:33:33,284] {standard_task_runner.py:52} INFO - Started process 145 to run task
[2021-12-27 11:33:34,017] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_kildare.scrap_data scheduled__2021-12-26T09:45:00+00:00 [running]> on host 0c125a5bba95
[2021-12-27 11:33:35,301] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clement.liscoet@gmail.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_kildare
AIRFLOW_CTX_TASK_ID=scrap_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-26T09:45:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-26T09:45:00+00:00
[2021-12-27 11:33:35,433] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://spark:7077, Port: None, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-27 11:33:35,446] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py kildare
[2021-12-27 11:34:28,756] {spark_submit.py:514} INFO - Using properties file: null
[2021-12-27 11:34:30,855] {spark_submit.py:514} INFO - WARNING: An illegal reflective access operation has occurred
[2021-12-27 11:34:30,857] {spark_submit.py:514} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.6/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-12-27 11:34:30,860] {spark_submit.py:514} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-12-27 11:34:30,862] {spark_submit.py:514} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-12-27 11:34:30,865] {spark_submit.py:514} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-12-27 11:34:31,340] {spark_submit.py:514} INFO - Parsed arguments:
[2021-12-27 11:34:31,341] {spark_submit.py:514} INFO - master                  spark://spark:7077
[2021-12-27 11:34:31,343] {spark_submit.py:514} INFO - deployMode              null
[2021-12-27 11:34:31,345] {spark_submit.py:514} INFO - executorMemory          null
[2021-12-27 11:34:31,347] {spark_submit.py:514} INFO - executorCores           null
[2021-12-27 11:34:31,350] {spark_submit.py:514} INFO - totalExecutorCores      null
[2021-12-27 11:34:31,353] {spark_submit.py:514} INFO - propertiesFile          null
[2021-12-27 11:34:31,355] {spark_submit.py:514} INFO - driverMemory            null
[2021-12-27 11:34:31,358] {spark_submit.py:514} INFO - driverCores             null
[2021-12-27 11:34:31,360] {spark_submit.py:514} INFO - driverExtraClassPath    null
[2021-12-27 11:34:31,363] {spark_submit.py:514} INFO - driverExtraLibraryPath  null
[2021-12-27 11:34:31,367] {spark_submit.py:514} INFO - driverExtraJavaOptions  null
[2021-12-27 11:34:31,372] {spark_submit.py:514} INFO - supervise               false
[2021-12-27 11:34:31,376] {spark_submit.py:514} INFO - queue                   null
[2021-12-27 11:34:31,378] {spark_submit.py:514} INFO - numExecutors            null
[2021-12-27 11:34:31,382] {spark_submit.py:514} INFO - files                   null
[2021-12-27 11:34:31,384] {spark_submit.py:514} INFO - pyFiles                 null
[2021-12-27 11:34:31,385] {spark_submit.py:514} INFO - archives                null
[2021-12-27 11:34:31,387] {spark_submit.py:514} INFO - mainClass               null
[2021-12-27 11:34:31,388] {spark_submit.py:514} INFO - primaryResource         file:/opt/***/jobs/get_ads_per_county.py
[2021-12-27 11:34:31,390] {spark_submit.py:514} INFO - name                    scrap data
[2021-12-27 11:34:31,392] {spark_submit.py:514} INFO - childArgs               [kildare]
[2021-12-27 11:34:31,393] {spark_submit.py:514} INFO - jars                    null
[2021-12-27 11:34:31,394] {spark_submit.py:514} INFO - packages                null
[2021-12-27 11:34:31,396] {spark_submit.py:514} INFO - packagesExclusions      null
[2021-12-27 11:34:31,398] {spark_submit.py:514} INFO - repositories            null
[2021-12-27 11:34:31,399] {spark_submit.py:514} INFO - verbose                 true
[2021-12-27 11:34:31,401] {spark_submit.py:514} INFO - 
[2021-12-27 11:34:31,403] {spark_submit.py:514} INFO - Spark properties used, including those specified through
[2021-12-27 11:34:31,405] {spark_submit.py:514} INFO - --conf and those from the properties file null:
[2021-12-27 11:34:31,407] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-27 11:34:31,409] {spark_submit.py:514} INFO - 
[2021-12-27 11:34:31,412] {spark_submit.py:514} INFO - 
[2021-12-27 11:34:36,660] {spark_submit.py:514} INFO - Main class:
[2021-12-27 11:34:36,666] {spark_submit.py:514} INFO - org.apache.spark.deploy.PythonRunner
[2021-12-27 11:34:36,671] {spark_submit.py:514} INFO - Arguments:
[2021-12-27 11:34:36,677] {spark_submit.py:514} INFO - file:/opt/***/jobs/get_ads_per_county.py
[2021-12-27 11:34:36,680] {spark_submit.py:514} INFO - null
[2021-12-27 11:34:36,685] {spark_submit.py:514} INFO - kildare
[2021-12-27 11:34:36,692] {spark_submit.py:514} INFO - --verbose
[2021-12-27 11:34:36,699] {spark_submit.py:514} INFO - Spark config:
[2021-12-27 11:34:36,706] {spark_submit.py:514} INFO - (spark.app.name,scrap data)
[2021-12-27 11:34:36,712] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-27 11:34:36,719] {spark_submit.py:514} INFO - (spark.submit.pyFiles,)
[2021-12-27 11:34:36,724] {spark_submit.py:514} INFO - (spark.submit.deployMode,client)
[2021-12-27 11:34:36,729] {spark_submit.py:514} INFO - Classpath elements:
[2021-12-27 11:34:36,735] {spark_submit.py:514} INFO - 
[2021-12-27 11:34:36,740] {spark_submit.py:514} INFO - 
[2021-12-27 11:34:36,745] {spark_submit.py:514} INFO - 
[2021-12-27 11:35:16,580] {spark_submit.py:514} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2021-12-27 11:35:16,828] {spark_submit.py:514} INFO - 21/12/27 11:35:16 INFO SparkContext: Running Spark version 3.2.0
[2021-12-27 11:35:17,522] {spark_submit.py:514} INFO - 21/12/27 11:35:17 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2021-12-27 11:35:21,581] {spark_submit.py:514} INFO - 21/12/27 11:35:21 INFO ResourceUtils: ==============================================================
[2021-12-27 11:35:21,598] {spark_submit.py:514} INFO - 21/12/27 11:35:21 INFO ResourceUtils: No custom resources configured for spark.driver.
[2021-12-27 11:35:21,602] {spark_submit.py:514} INFO - 21/12/27 11:35:21 INFO ResourceUtils: ==============================================================
[2021-12-27 11:35:21,608] {spark_submit.py:514} INFO - 21/12/27 11:35:21 INFO SparkContext: Submitted application: Get Ads per county
[2021-12-27 11:35:23,190] {spark_submit.py:514} INFO - 21/12/27 11:35:23 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2021-12-27 11:35:23,407] {spark_submit.py:514} INFO - 21/12/27 11:35:23 INFO ResourceProfile: Limiting resource is cpu
[2021-12-27 11:35:23,408] {spark_submit.py:514} INFO - 21/12/27 11:35:23 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2021-12-27 11:35:24,826] {spark_submit.py:514} INFO - 21/12/27 11:35:24 INFO SecurityManager: Changing view acls to: ***
[2021-12-27 11:35:24,860] {spark_submit.py:514} INFO - 21/12/27 11:35:24 INFO SecurityManager: Changing modify acls to: ***
[2021-12-27 11:35:24,875] {spark_submit.py:514} INFO - 21/12/27 11:35:24 INFO SecurityManager: Changing view acls groups to:
[2021-12-27 11:35:24,876] {spark_submit.py:514} INFO - 21/12/27 11:35:24 INFO SecurityManager: Changing modify acls groups to:
[2021-12-27 11:35:24,889] {spark_submit.py:514} INFO - 21/12/27 11:35:24 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(***); groups with view permissions: Set(); users  with modify permissions: Set(***); groups with modify permissions: Set()
[2021-12-27 11:35:36,787] {spark_submit.py:514} INFO - 21/12/27 11:35:36 INFO Utils: Successfully started service 'sparkDriver' on port 43633.
[2021-12-27 11:35:43,518] {spark_submit.py:514} INFO - 21/12/27 11:35:43 INFO SparkEnv: Registering MapOutputTracker
[2021-12-27 11:35:48,235] {spark_submit.py:514} INFO - 21/12/27 11:35:48 INFO SparkEnv: Registering BlockManagerMaster
[2021-12-27 11:35:48,685] {spark_submit.py:514} INFO - 21/12/27 11:35:48 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2021-12-27 11:35:48,745] {spark_submit.py:514} INFO - 21/12/27 11:35:48 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2021-12-27 11:35:49,214] {spark_submit.py:514} INFO - 21/12/27 11:35:49 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2021-12-27 11:35:53,125] {spark_submit.py:514} INFO - 21/12/27 11:35:53 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-43d6f06c-53d7-4e8e-8714-8fc3e730d4db
[2021-12-27 11:35:53,896] {spark_submit.py:514} INFO - 21/12/27 11:35:53 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2021-12-27 11:35:54,945] {spark_submit.py:514} INFO - 21/12/27 11:35:54 INFO SparkEnv: Registering OutputCommitCoordinator
[2021-12-27 11:36:22,939] {spark_submit.py:514} INFO - 21/12/27 11:36:22 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2021-12-27 11:36:22,970] {spark_submit.py:514} INFO - 21/12/27 11:36:22 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2021-12-27 11:36:22,973] {spark_submit.py:514} INFO - 21/12/27 11:36:22 WARN Utils: Service 'SparkUI' could not bind on port 4042. Attempting port 4043.
[2021-12-27 11:36:22,977] {spark_submit.py:514} INFO - 21/12/27 11:36:22 WARN Utils: Service 'SparkUI' could not bind on port 4043. Attempting port 4044.
[2021-12-27 11:36:22,984] {spark_submit.py:514} INFO - 21/12/27 11:36:22 WARN Utils: Service 'SparkUI' could not bind on port 4044. Attempting port 4045.
[2021-12-27 11:36:22,988] {spark_submit.py:514} INFO - 21/12/27 11:36:22 WARN Utils: Service 'SparkUI' could not bind on port 4045. Attempting port 4046.
[2021-12-27 11:36:22,991] {spark_submit.py:514} INFO - 21/12/27 11:36:22 WARN Utils: Service 'SparkUI' could not bind on port 4046. Attempting port 4047.
[2021-12-27 11:36:22,996] {spark_submit.py:514} INFO - 21/12/27 11:36:22 WARN Utils: Service 'SparkUI' could not bind on port 4047. Attempting port 4048.
[2021-12-27 11:36:23,000] {spark_submit.py:514} INFO - 21/12/27 11:36:22 WARN Utils: Service 'SparkUI' could not bind on port 4048. Attempting port 4049.
[2021-12-27 11:36:23,003] {spark_submit.py:514} INFO - 21/12/27 11:36:22 WARN Utils: Service 'SparkUI' could not bind on port 4049. Attempting port 4050.
[2021-12-27 11:36:23,022] {spark_submit.py:514} INFO - 21/12/27 11:36:22 WARN Utils: Service 'SparkUI' could not bind on port 4050. Attempting port 4051.
[2021-12-27 11:36:23,034] {spark_submit.py:514} INFO - 21/12/27 11:36:22 WARN Utils: Service 'SparkUI' could not bind on port 4051. Attempting port 4052.
[2021-12-27 11:36:23,051] {spark_submit.py:514} INFO - 21/12/27 11:36:22 WARN Utils: Service 'SparkUI' could not bind on port 4052. Attempting port 4053.
[2021-12-27 11:36:23,208] {spark_submit.py:514} INFO - 21/12/27 11:36:23 INFO Utils: Successfully started service 'SparkUI' on port 4053.
[2021-12-27 11:36:27,422] {spark_submit.py:514} INFO - 21/12/27 11:36:27 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://0c125a5bba95:4053
[2021-12-27 11:36:37,662] {spark_submit.py:514} INFO - 21/12/27 11:36:37 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-27 11:36:42,390] {spark_submit.py:514} INFO - 21/12/27 11:36:42 INFO TransportClientFactory: Successfully created connection to spark/172.19.0.4:7077 after 2461 ms (0 ms spent in bootstraps)
[2021-12-27 11:36:46,295] {spark_submit.py:514} INFO - 21/12/27 11:36:46 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20211227113645-0003
[2021-12-27 11:36:48,045] {spark_submit.py:514} INFO - 21/12/27 11:36:48 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 33999.
[2021-12-27 11:36:48,048] {spark_submit.py:514} INFO - 21/12/27 11:36:48 INFO NettyBlockTransferService: Server created on 0c125a5bba95:33999
[2021-12-27 11:36:48,054] {spark_submit.py:514} INFO - 21/12/27 11:36:48 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2021-12-27 11:36:48,930] {spark_submit.py:514} INFO - 21/12/27 11:36:48 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 0c125a5bba95, 33999, None)
[2021-12-27 11:36:49,656] {spark_submit.py:514} INFO - 21/12/27 11:36:49 INFO BlockManagerMasterEndpoint: Registering block manager 0c125a5bba95:33999 with 434.4 MiB RAM, BlockManagerId(driver, 0c125a5bba95, 33999, None)
[2021-12-27 11:36:49,777] {spark_submit.py:514} INFO - 21/12/27 11:36:49 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 0c125a5bba95, 33999, None)
[2021-12-27 11:36:51,039] {spark_submit.py:514} INFO - 21/12/27 11:36:51 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 0c125a5bba95, 33999, None)
[2021-12-27 11:37:22,345] {spark_submit.py:514} INFO - 21/12/27 11:37:22 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2021-12-27 11:37:25,459] {spark_submit.py:514} INFO - /home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py:238: FutureWarning: Python 3.6 support is deprecated in Spark 3.2.
[2021-12-27 11:37:25,465] {spark_submit.py:514} INFO - FutureWarning
[2021-12-27 11:38:03,440] {spark_submit.py:514} INFO - 21/12/27 11:38:03 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2021-12-27 11:38:05,294] {spark_submit.py:514} INFO - 21/12/27 11:38:05 INFO SharedState: Warehouse path is 'file:/opt/***/spark-warehouse'.
[2021-12-27 11:38:43,458] {spark_submit.py:514} INFO - 21/12/27 11:38:43 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20211227113645-0003/0 on worker-20211227113155-172.19.0.5-38589 (172.19.0.5:38589) with 6 core(s)
[2021-12-27 11:38:43,473] {spark_submit.py:514} INFO - 21/12/27 11:38:43 INFO StandaloneSchedulerBackend: Granted executor ID app-20211227113645-0003/0 on hostPort 172.19.0.5:38589 with 6 core(s), 1024.0 MiB RAM
[2021-12-27 11:38:43,498] {spark_submit.py:514} INFO - 21/12/27 11:38:43 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20211227113645-0003/1 on worker-20211227113155-172.19.0.6-41167 (172.19.0.6:41167) with 6 core(s)
[2021-12-27 11:38:43,521] {spark_submit.py:514} INFO - 21/12/27 11:38:43 INFO StandaloneSchedulerBackend: Granted executor ID app-20211227113645-0003/1 on hostPort 172.19.0.6:41167 with 6 core(s), 1024.0 MiB RAM
[2021-12-27 11:38:43,877] {spark_submit.py:514} INFO - 21/12/27 11:38:43 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20211227113645-0003/0 is now RUNNING
[2021-12-27 11:38:43,880] {spark_submit.py:514} INFO - 21/12/27 11:38:43 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20211227113645-0003/1 is now RUNNING
[2021-12-27 11:38:51,419] {spark_submit.py:514} INFO - running...
[2021-12-27 11:38:55,662] {spark_submit.py:514} INFO - running...
[2021-12-27 11:38:57,526] {spark_submit.py:514} INFO - running...
[2021-12-27 11:38:59,611] {spark_submit.py:514} INFO - running...
[2021-12-27 11:39:00,493] {spark_submit.py:514} INFO - 21/12/27 11:39:00 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Registered executor NettyRpcEndpointRef(spark-client://Executor) (172.19.0.6:34576) with ID 1,  ResourceProfileId 0
[2021-12-27 11:39:00,504] {spark_submit.py:514} INFO - 21/12/27 11:39:00 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Registered executor NettyRpcEndpointRef(spark-client://Executor) (172.19.0.5:58246) with ID 0,  ResourceProfileId 0
[2021-12-27 11:39:01,061] {spark_submit.py:514} INFO - running...
[2021-12-27 11:39:03,145] {spark_submit.py:514} INFO - running...
[2021-12-27 11:39:32,222] {taskinstance.py:1703} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py kildare. Error code is: -9.
[2021-12-27 11:39:32,796] {taskinstance.py:1280} INFO - Marking task as UP_FOR_RETRY. dag_id=scraping_daft_kildare, task_id=scrap_data, execution_date=20211226T094500, start_date=20211227T113332, end_date=20211227T113932
[2021-12-27 11:39:35,165] {standard_task_runner.py:91} ERROR - Failed to execute job 4808 for task scrap_data
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 292, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py kildare. Error code is: -9.
[2021-12-27 11:39:38,319] {local_task_job.py:154} INFO - Task exited with return code 1
[2021-12-27 11:39:39,587] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
